src/api/tests/sql_test_misc.cc

```diff
@@ -0,0 +1,351 @@
+               StringUtils::trim(std::string(fields[1])) + "', '" +
+               StringUtils::trim(std::string(fields[2])) + "', '" +
+               StringUtils::trim(std::string(fields[3])) + "');\n";
+    }
+    query += "COMMIT;";
+    hustle_db->execute_query_result(query);
+    std::cerr << "part done" << std::endl;
+
+    stream = fopen(SUPPLIER_PATH, "r");
+    query = "BEGIN TRANSACTION;";
+    while (fgets(line, 2048, stream)) {
+      char* tmp = strdup(line);
+      // std::cout << line << std::endl;
+      char** fields = getfields(tmp, 4);
+      query += "INSERT INTO supplier VALUES (" +
+               StringUtils::trim(std::string(fields[0])) + ", '" +
+               StringUtils::trim(std::string(fields[1])) + "', '" +
+               StringUtils::trim(std::string(fields[2])) + "', '" +
+               StringUtils::trim(std::string(fields[3])) + "');\n";
+    }
+    query += "COMMIT;";
+    hustle_db->execute_query_result(query);
+    std::cerr << "supplier done" << std::endl;
+
+    stream = fopen(CUSTOMER_PATH, "r");
+    query = "BEGIN TRANSACTION;";
+    while (fgets(line, 2048, stream)) {
+      char* tmp = strdup(line);
+      char** fields = getfields(tmp, 17);
+      query += "INSERT INTO customer VALUES (" +
+               StringUtils::trim(std::string(fields[0])) + ", '" +
+               StringUtils::trim(std::string(fields[1])) + "', '" +
+               StringUtils::trim(std::string(fields[2])) + "', '" +
+               StringUtils::trim(std::string(fields[3])) + "');\n";
+    }
+    query += "COMMIT;";
+    hustle_db->execute_query_result(query);
+    std::cerr << "customer done" << std::endl;
+
+    stream = fopen(DATE_PATH, "r");
+    query = "BEGIN TRANSACTION;";
+    while (fgets(line, 2048, stream)) {
+      char* tmp = strdup(line);
+      char** fields = getfields(tmp, 7);
+      query += "INSERT INTO ddate VALUES (" +
+               StringUtils::trim(std::string(fields[0])) + ", '" +
+               StringUtils::trim(std::string(fields[1])) + "', '" +
+               StringUtils::trim(std::string(fields[2])) + "'," +
+               StringUtils::trim(std::string(fields[3])) + ", " +
+               StringUtils::trim(std::string(fields[4])) + ", " +
+               StringUtils::trim(std::string(fields[5])) + "," + "'" +
+               StringUtils::trim(std::string(fields[6])) + "');\n";
+    }
+    query += "COMMIT;";
+    hustle_db->execute_query_result(query);
+    std::cerr << "date done" << std::endl;
+
+    hustle::HustleDB::start_scheduler();
+  }
+
+  void TearDown() override { hustle::HustleDB::stop_scheduler(); }
+};
+
+hustle::catalog::TableSchema SQLTest::part("part"),
+    SQLTest::supplier("supplier"), SQLTest::customer("customer"),
+    SQLTest::ddate("ddate"), SQLTest::lineorder("lineorder");
+
+DBTable::TablePtr SQLTest::lo, SQLTest::d, SQLTest::p, SQLTest::c, SQLTest::s;
+
+std::shared_ptr<hustle::HustleDB> SQLTest::hustle_db;
+
+TEST_F(SQLTest, q1_join_reorder) {
```

Comment:
A good practice is to use descriptive test names, so a reader can easily identify what the test is testing. Here this test is not testing a reorder but rather the correctness of a query result